Sources/BraintreeThreeDSecure/Models/BTThreeDSecureAuthenticateJWTPOSTBody.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+// The POST body for /three_d_secure/authenticate_from_jwt
+struct BTThreeDSecureAuthenticateJWTPOSTBody: Encodable {
+    
+    // MARK: - Private Properties
+    
+    private let jwt: String
+    private let paymentMethodNonce: String
+    
+    enum CodingKeys: String, CodingKey {
+        case jwt
+        case paymentMethodNonce
+    }
```

Comment:
Do we need a CodingKeys enum here if we aren't setting any custom key strings?

Reply:
addressed in d84e7506f4784e792f1996857fc82021d96376bb